Traceback (most recent call last):
  File "/CWD/crashing_test.py", line 1, in <module>
    raise Exception("Nothing to see here")
Exception: Nothing to see here
